engine/src/test/java/io/camunda/zeebe/engine/processing/deployment/model/validation/TimerValidationTest.java

```diff
@@ -35,7 +35,7 @@ void invalidCycleFormat(
         process,
         expect(
             timerEventElementId,
-            "Invalid timer cycle expression (Repetition spec must start with R)"));
+            "Invalid timer cycle expression (Cron expression must consist of 6 fields (found 1 in \"foo\"))"));
```

Comment:
❌ We should let the user know that the invalid time cycle expression is invalid because it is not a correct Repetition spec **and** not a correct CRON expression